vars/tagGit.groovy

```diff
@@ -31,7 +31,11 @@ def call(Map <String, ?> config = [:]) {
       ) {
         node(POD_LABEL) {
           container('git') {
-
+            
+            if(config.addSSHKey){
+              sh "ssh-keyscan -t ecdsa github.com >> ~/.ssh/known_hosts"
```

Comment:
could we also introduce an `ssh_key_type` parameter, so that someone can add both rsa and ecdsa, etc. keys?

Reply:
Sure!

Reply:
Should be done now, although from your wording I took it to mean that these params should not be optional, so if that needs to be changed let me know